scripts/sct_analyze_texture.py

```diff
@@ -23,7 +23,7 @@
 import sct_utils as sct
 import spinalcordtoolbox.image as msct_image
 from spinalcordtoolbox.image import Image
-from spinalcordtoolbox.utils import Metavar
+from spinalcordtoolbox.utils import Metavar, SmartFormatter
```

Comment:
@YangHee-Min you also need to use `SmartFormatter` when initiating `ArgumentParser`

Reply:
Oops my mistake. Added in commit [93e3e46](https://github.com/neuropoly/spinalcordtoolbox/commit/93e3e46cab6db3c42198a4cdf26bc5e6144527ac)